update-image.sh

```diff
@@ -20,9 +21,14 @@ echo -e "Creating ECR repository..."
 aws ecr create-repository --repository-name $LOGICAL_NAME || true
 
 echo -e "Tagging Image..."
-docker tag $LOGICAL_NAME:latest $ECR_URI:latest
+docker tag $LOGICAL_NAME:$TAG $ECR_URI:$TAG
 
 echo -e "Pushing to ECR..."
-docker push $ECR_URI:latest
+docker push $ECR_URI:$TAG
 
-aws ecs update-service --force-new-deployment --cluster repost --service $LOGICAL_NAME || true
+# Does this actually belong here? The scope of this action should be to simply push images to ECR,
+# it should not really be redeploying ECS services automatically when the image is updated.
```

Comment:
calling this out for discussion...

Reply:
I updated the workflow steps in core to be clear that it's not just publishing the artifact but also deploying. I don't think it belongs here, should be a separate action and step in the workflows.